### ISSUE TITLE
Training and Validation accuracy

### ISSUE THREAD

**Issue by Dishank99** [JamesChuanggg/VQG-tensorflow#7]
Hi, how can I get training accuracy and  validation accuracy for your model
